### ISSUE TITLE
[Bug]: The sidebar overlaps the content

### ISSUE THREAD

**Issue by victorlunam** [reactjs/react.dev#6941]
### userAgent
```
# navigator.userAgent
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
```

### screen sizes
* height: 824
* width: 1445

### issue

the sidebar is displayed on top of the content, the same problem occurs on different pages

<img width="1128" alt="image" src="https://github.com/reactjs/react.dev/assets/98234382/42ef36e6-fb64-4566-8b82-345f955289e9">
<img width="1039" alt="image" src="https://github.com/reactjs/react.dev/assets/98234382/44708c3c-2b99-4e94-a1ed-e31d61903216">

### Page

https://react.dev/learn

### Details

some issues related:

* https://github.com/reactjs/react.dev/issues/5588
* https://github.com/reactjs/react.dev/pull/5591

**Comment by tusharagrawall** [comment]
give me the honour of resolving this bug. ill do this within few minutes. @victorlunam

**Comment by victorlunam** [comment]
> give me the honour of resolving this bug. ill do this within few minutes. @victorlunam

of course, could resolve this issue. for my part I made a PR, offering a possible solution https://github.com/reactjs/react.dev/pull/6943.

see the result [here](https://react-dev-git-fork-victorlunam-fix-sidebarn-87059f-fbopensource.vercel.app/)

**Comment by victorlunam** [comment]
> I think you can adjust your browser default font size to 16px or larger.

I think that adjusting the default font size or changing some similar setting will never be the solution, considering that the screen size is the standard one.
